### ISSUE TITLE
Mode to run indefinitely

### ISSUE THREAD

**Issue by mrjackdavis** [Trioxis/aws-backup-manager#41]
To make this more immediately useful, it would be nice if the script could run in a mode where it would run regularly. Perhaps even setting the frequency, and length of time it should run.

    $ npm start --length indefinite --frequency 6000

**Comment by Dermah** [comment]
This pleases me

**Comment by mrjackdavis** [comment]
Would you use something like [`forever`](https://www.npmjs.com/package/forever)? It should be pretty easy really.

**Comment by Dermah** [comment]
Actually I think this should be done in the parent repository, since this repo will soon become just the engine that backs up volumes to snapshots. It is the responsibility of the parent repository to run this thing forever/when it wants.

**Comment by mrjackdavis** [comment]
@Dermah I agree, if there is a parent orchestrator, it should manage the scripts. But there could be a circumstance where an admin doesn't want to setup the orchestration. Do we want to support that case?
